servers/quarkus-server/src/main/java/org/projectnessie/server/authz/CompiledAuthorizationRules.java

```diff
@@ -81,6 +87,26 @@ public CompiledAuthorizationRules(QuarkusNessieAuthorizationConfig config) {
     return ImmutableMap.copyOf(scripts);
   }
 
+  // TODO Quarkus/microprofile config API do not allow maps. Using this workaround here until
+  //  https://github.com/quarkusio/quarkus/issues/19990 is fixed.
+  private static Map<String, String> rulesFromConfig() {
+    String prefix = "nessie.server.authorization.rules";
+    Config config = ConfigProvider.getConfig();
+    return StreamSupport.stream(config.getPropertyNames().spliterator(), false)
+        .filter(name -> name.startsWith(prefix) && !name.equalsIgnoreCase(prefix))
+        .collect(
+            Collectors.toMap(
+                prop -> cleanPropertyName(prop.substring(prefix.length() + 1)),
+                prop -> config.getOptionalValue(prop, String.class).orElse("")));
+  }
+
+  private static String cleanPropertyName(String name) {
+    if (name.startsWith("\"") && name.endsWith("\"")) {
```

Comment:
How can quotes occur in this context, given that `name` is a substring of a quarkus property?

Reply:
TIL :)